tests/typowalking/wordwalker.js

```diff
@@ -153,10 +153,42 @@ bender.test( {
 		// although we have solved the problem of inner list being walked twice,
 		// it's not smart enough yet to realize we need to add a whitespace when skipping over.
 		// we need to detect this special case and add a whitespace (harder than it sounds)
-		arrayAssert.itemsAreEqual(['foobaz'], this.getWordsWithWordWalker( outerUnorderedList.getFirst() ));
+		arrayAssert.itemsAreEqual(['foo', 'baz'], this.getWordsWithWordWalker( outerUnorderedList.getFirst() ));
```

Comment:
This is to make the test actually fail given that we don't run CI on this, and it's better to have the test reflect intended reality (as described in the comment above)